services/movie.py

```diff
@@ -1,11 +1,12 @@
 from django.db.models import QuerySet
-
+from django.db import transaction
 from db.models import Movie
 
 
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
```

Comment:
Using mutable default arguments like `list[int]` can lead to unexpected behavior. It's safer to use `None` as a default value and initialize the list inside the function: `genres_ids: list[int] = None` and then `if genres_ids is None: genres_ids = []`.